docs/Installation.md

```diff
@@ -1,8 +1,137 @@
-# Installation of AppStudio E2E Mode in OpenShift CI
+# RHTAP Installation locally and Openshift CI
 
-The following guide will walk through the deployment of AppStudio E2E mode in Openshift CI Pull Requests jobs
+The following guide will walk through the necessary instruction about driving an installation of RHTAP locally or in Openshift CI.
 
-## Install e2e binary in openshift-ci and use pairing PRs feature
+## RHTAP in dev mode
+
+
+### Requirements
+
+Requirements for installing RHTAP in E2E mode:
+
+* An OpenShift 4.12 or higher Environment (If you are using CRC/OpenShift Local please also review [optional-codeready-containers-post-bootstrap-configuration](https://github.com/redhat-appstudio/infra-deployments/blob/main/docs/development/deployment.md#optional-openshift-local-post-bootstrap-configuration))
+* A machine from which to run the install (usually your laptop) with required tools:
+  * A properly setup Go workspace using **Go 1.19 is required**
+  * The OpenShift Command Line Tool (oc) **Use the version coresponding to the Openshift version**
+  * yq
+  * [jq](https://github.com/mikefarah/yq)
+  * git
+  * helm
+  * vcluster (!optional! Only in case of running byoc testing)
+* Tokens
+  * Github Token with the following permissions
+    * `repo`
+    * `delete_repo`
+  * Valid quay token where to push RHTAP components images generated by the e2e framework
+
+### Installation
+
+1. Before deploying RHTAP in E2E mode you need to login to your OpenShift cluster with OpenShift Command Line Tool as `admin` (by default  `kubeadmin`):
+
+   ```bash
+    oc login -u <user> -p <password> --server=<oc_api_url>
+   ```
+
+2. Export required (and recommended) environment variables from [default.env](../default.env). Copy the file (`cp default.env user.env`), edit the required variables and source it (`source user.env`).
+
+3. Install dependencies:
+
+``` bash
+# Install dependencies
+$ go mod tidy
+# or go mod tidy -compat=1.19
+# Copy the dependencies to vendor folder
+$ go mod vendor
+```
+
+4. By default the installation script will use the `redhat-appstudio-qe` GitHub organization for pushing changes to `infra-deployments` repository.
+
+**It is recommended to use your fork of [infra-deployments repo](https://github.com/redhat-appstudio/infra-deployments) in your GitHub org instead** - you can change the GitHub organization with environment variable `export MY_GITHUB_ORG=<name-of-your-github-org>`.
+
+<details>
+<summary><h4>Advanced installation guidelines</h4></summary>
+<h5>GitHub Application for Pipelines as Code<h5>
+<p>
+   Some tests could require you have a Github App created in order to test Component builds via Pipelines as Code.
+Such tests are [rhtap-demo](https://github.com/redhat-appstudio/e2e-tests/blob/main/tests/rhtap-demo/rhtap-demo.go) and [build](https://github.com/redhat-appstudio/e2e-tests/blob/main/tests/build/build.go).
```

Comment:
these 2 links are not rendered as hyperlinks, I suppose that's not intentional?
![image](https://github.com/redhat-appstudio/e2e-tests/assets/43273420/37e3addc-b95b-4081-bf08-1b19cecb0341)

Reply:
is html :)

Reply:
Tested locally, if we add a new line before line 56, it will correctly render as a hyperlink.
But I'll not block the PR due to this.